misk-aws/src/test/kotlin/misk/jobqueue/sqs/SqsJobQueueTest.kt

```diff
@@ -90,10 +98,10 @@ internal class SqsJobQueueTest {
     )
 
     // Confirm metrics
-    assertThat(sqsMetrics.jobsEnqueued.labels(queueName.value).get()).isEqualTo(10.0)
+    assertThat(sqsMetrics.jobsEnqueued.labels(queueName.value).get()).isEqualTo(20.0)
     assertThat(sqsMetrics.jobEnqueueFailures.labels(queueName.value).get()).isEqualTo(0.0)
-    assertThat(sqsMetrics.jobsReceived.labels(queueName.value).get()).isEqualTo(10.0)
-    assertThat(sqsMetrics.jobsAcknowledged.labels(queueName.value).get()).isEqualTo(10.0)
+    assertThat(sqsMetrics.jobsReceived.labels(queueName.value).get()).isEqualTo(20.0)
+    assertThat(sqsMetrics.jobsAcknowledged.labels(queueName.value).get()).isEqualTo(20.0)
```

Comment:
ah this makes sense. SQS would double count the metrics and the app itself will be responsible for filtering